app/models/incident.rb

```diff
@@ -95,25 +95,31 @@ def claim_for(user)
   end
 
   def export_to_claims!
-    return unless completed?
-    begin
-      ci = ClaimsIncident.create claims_fields table: :incident
-      update claims_id: ci.UID
-      ClaimsDriversReport.create claims_fields table: :drivers_report
-    rescue ActiveRecord::StatementInvalid => e
-      puts e.cause and return false
-      # TODO: report failure to the user, and report error to programmers
-      # We only get here if an error was caused by a programmer.
-      # The constraints on completed records should be such that we never reach this rescue block.
+    if completed? && valid?
```

Comment:
Just because things were valid before doesn't mean they still are. We should check again before exporting.